CONTRIBUTING.md

```diff
@@ -47,7 +47,7 @@ You need the following software to develop temBoard:
 - bash, git, make, psql.
 - Docker Compose v2.
 - Python 3.6 with `venv` module.
-- NodeJS 16+ and npm for building some browser assets.
+- NodeJS 20+ and npm for building some browser assets.
```

Comment:
We should force the version of Node in the package.json file.

https://stackoverflow.com/questions/29349684/how-can-i-specify-the-required-node-js-version-in-package-json